classes/class-wc-connect-nux.php

```diff
@@ -186,11 +217,14 @@ public function set_up_nux_notices() {
 			}
 
 			$jetpack_install_status = $this->get_jetpack_install_status();
+			$banner_to_display = $this->get_banner_type_to_display( array(
```

Comment:
What do you think about merging `get_jetpack_install_status()` into `get_banner_type_to_display()`?

I think it'll make the code easier to read:

```
if ( ! class_exists() ) return 'before_jetpack_connection';
```